tools/sgen/output/FluentUIStyle.generated.swift

```diff
@@ -891,6 +891,118 @@ open class FluentUIStyle: NSObject {
 		}
 
 	}
+	// MARK: - MSFDangerToastNotificationTokens
+	open var MSFDangerToastNotificationTokens: MSFDangerToastNotificationTokensAppearanceProxy {
```

Comment:
Am I supposed to include these output files in the change? Is that what's causing the validation issues?